spec/features/officing/booth_spec.rb

```diff
@@ -73,9 +79,9 @@
     ba2 = create(:poll_booth_assignment, poll: poll2, booth: booth2)
     ba3 = create(:poll_booth_assignment, poll: poll2, booth: booth2)
 
-    create(:poll_officer_assignment, officer: officer, booth_assignment: ba1, date: Date.today)
-    create(:poll_officer_assignment, officer: officer, booth_assignment: ba2, date: Date.today)
-    create(:poll_officer_assignment, officer: officer, booth_assignment: ba3, date: Date.today)
+    create(:poll_officer_assignment, officer: officer, booth_assignment: ba1, date: '2018-01-01')
+    create(:poll_officer_assignment, officer: officer, booth_assignment: ba2, date: '2018-01-01')
+    create(:poll_officer_assignment, officer: officer, booth_assignment: ba3, date: '2018-01-01')
```

Comment:
Same doubt as in the other related PRs...can't `Date.today` be used to define the `date` attribute?

Reply:
Yes, I didn't realised before. I've changed it.